app/models/drug_stock.rb

```diff
@@ -0,0 +1,9 @@
+class DrugStock < ApplicationRecord
+  belongs_to :facility
+  belongs_to :user
+  belongs_to :protocol_drug
+
+  validates :in_stock, numericality: true, presence: true
+  validates :received, numericality: true
+  validates :recorded_at, presence: true
```

Comment:
Should this go into `ApplicationRecord`?

Reply:
Resolved after talking to @vkrmis. Not all application records are "recorded", like `Facility`.